mijia-homie/src/main.rs

```diff
@@ -235,8 +192,7 @@ async fn run_sensor_system(
     mut homie: HomieDevice,
     session: &MijiaSession,
 ) -> Result<(), eyre::Report> {
-    let sensor_names = hashmap_from_file(SENSOR_NAMES_FILENAME)
-        .wrap_err(format!("reading {}", SENSOR_NAMES_FILENAME))?;
+    let sensor_names = read_sensor_names()?;
```

Comment:
suddenly feels odd that this isn't passed in from the top

Reply:
Done.